rust/common-wit/build.rs

```diff
@@ -9,6 +9,18 @@ const TYPESCRIPT_SOURCE_DEPENDENCIES: &[&str] = &[
 ];
 
 fn main() {
+    // Clean node_modules directory to avoid file conflicts
```

Comment:
This shouldn't be necessary I think - `npm ci` should be a clean install - https://docs.npmjs.com/cli/v10/commands/npm-ci

> If a node_modules is already present, it will be automatically removed before npm ci begins its install.

Reply:
for the paper trail, I believe it's the clean itself that is failing to clean up the symlinked file https://github.com/commontoolsinc/system/actions/runs/10058226982/job/27801092879#step:12:23